pytorch_lightning/profiler/pytorch.py

```diff
@@ -90,7 +90,7 @@ def __enter__(self) -> None:
 
                 self._handles[module_name] = [pre_forward_handle, post_forward_handle]
 
-    def __exit__(self, exc_type: Any, exc_value: Any, traceback: Any) -> None:
+    def __exit__(self, type: Any, value: Any, traceback: Any) -> None:
```

Comment:
type is a built-in keyword. 
better to avoid name clash here

Reply:
Vulture picks up `exc_type` because it doesn't match the parent signature. We have a few options:
- leave like this, matches the parent
- convert to `_`, `__`, `___` - standard for unused variables
- add to whitelist (already a bit big)